tools/polarshift.rb

```diff
@@ -379,6 +396,60 @@ def setup_global_opts(options)
       end
       @opts = opts
     end
+
+    # given a query_result from a call to get_smart_run()
+    # @return Array of testcase IDs
+    def extract_test_case_ids(query_result)
+      query_result['records']['TestRecord'].map { |tr| tr['test_case']['id'] }
+    end
+
+    # given a query_result Custom object (Array)
+    # @return Hash to be constructed
```

Comment:
could you clarify this? Maybe provide a simple example in comment?
Other than these two, lg

Reply:
@akostadinov ok, updated.... PTAL again.  Thanks!